stair-carousel/script.js

```diff
@@ -0,0 +1,70 @@
+function moveToSelected(element) {
+  if (element == "next") {
+    var selected = $(".selected").next();
+  } else if (element == "prev") {
+    var selected = $(".selected").prev();
+  } else {
+    var selected = element;
+  }
+
+  var next = $(selected).next();
+  var prev = $(selected).prev();
+  var prevSecond = $(prev).prev();
+  var nextSecond = $(next).next();
+
+  $(selected).removeClass().addClass("selected");
+
+  $(prev).removeClass().addClass("prev");
+  $(next).removeClass().addClass("next");
+
+  $(nextSecond).removeClass().addClass("nextSecond");
+  $(prevSecond).removeClass().addClass("prevSecond");
+
+  $(nextSecond).nextAll().removeClass().addClass("hideRight");
+  $(prevSecond).prevAll().removeClass().addClass("hideLeft");
+}
+
+function FindPos(obj) {
```

Comment:
Do we need this function? I think it is only printing a log to the console.
And, if needed, then any reasons to not use the camelCase naming convention here?